Info-Orbs/lib/config/config.h.template

```diff
@@ -1,87 +0,0 @@
-#ifndef CONFIG_H
```

Comment:
You don't want to delete this file

Reply:
It's not deleted, just renamed to config.h. By setting the WiFi credentials with WiFiManager they are no long stored in config.h.

Reply:
For the other settings it is still needed, and you did delete it according to git (because the config.h is not included in this commit (it is ignored due to .gitignore)